### DISCUSSION TITLE
Is the Israeli Shekel supported? ₪

### DISCUSSION THREAD

**Discussion by MartinDawson** [handsontable/hyperformula#694] [General]
I've tried lots of currencies, everything is working for me apart from the israeli shekel, currency symbol: ₪

It results in undefined.

I have checked hyperformula's unit tests but only see PLN and $ tested so I'm thinking this could have been missed?

I'm using this library for the currency string: https://www.npmjs.com/package/currency-symbol-map

This is on chrome latest and windows 10.

**Comment by izulin** [comment]
This is worth investigating (e.g. more exotic unicode inputs in config).

**Reply by MartinDawson** [reply]
It's not urgent at all for us but if there was a work around for it temporarily that we could do on our side in the meantime then that would be great.

I presume the only thing we could do in the meantime is to remove the currency symbol from hyperformula and then add it back to the displayed value after the calculation is done? Or is there any way we can manually register in the config the israeli symbol?

**Reply by izulin** [reply]
```
    const engine = HyperFormula.buildFromArray([['1₪']], {currencySymbol: ['₪']})
    expect(engine.getCellValueFormat({sheet: 0, row: 0, col: 0})).toEqual('₪')
    expect(engine.getCellValue({sheet: 0, row: 0, col: 0})).toEqual(1)
```

This code seems to be working for me.

Am I missing something?


edit: replace `adr` internal helper with proper address

**Reply by MartinDawson** [reply]
Yep, sorry. I was testing it incorrectly :/

This is all good. Shrekel works fine.
